external-builds/pytorch/patches/v2.7.0/pytorch/hipified/0006-Support-gfx1151-in-aotriton.patch

```diff
@@ -0,0 +1,67 @@
+From e4923146cbdf160c9ea292f115e147944a2a0768 Mon Sep 17 00:00:00 2001
+From: Aaryaman Vasishta <jem456.vasishta@gmail.com>
+Date: Mon, 31 Mar 2025 19:27:59 +0100
+Subject: [PATCH] Support gfx1151 in aotriton
+
+---
+ cmake/External/aotriton.cmake | 6 ++++--
+ cmake/public/LoadHIP.cmake    | 6 ++++++
+ 2 files changed, 10 insertions(+), 2 deletions(-)
+
+diff --git a/cmake/External/aotriton.cmake b/cmake/External/aotriton.cmake
+index 2678cfde3c..2ee7345d9d 100644
+--- a/cmake/External/aotriton.cmake
++++ b/cmake/External/aotriton.cmake
+@@ -2,6 +2,7 @@ macro(get_target_gpus_from_pytorch target_gpus)
+    set(gfx90a_key MI200)
+    set(gfx942_key MI300X)
+    set(gfx1100_key Navi31)
++   set(gfx1151_key Navi3.5)
```

Comment:
```suggestion
+   set(gfx1151_key Navi35)
```
I guess the naming scheme seems that this should rather be `Navi35` and not `Navi3.5` but I am aware that this implies more changes.

Reply:
This has to map with the corresponding name on `aotriton` itself. I'd keep it as navi35 just to avoid any potential issues with the dot when referring to as paths or within a search string etc.

Reply:
I inherited the "Navi3.5" name from @jammm's original patch and have no opinion regarding the dot.

I can go ahead and make the necessary changes in aotriton for "Navi35" if we have consensus and I don't have to change it back and forth ;)

Reply:
I don’t mind either way really ^^

Reply:
So we stick to Navi3.5 then?

Reply:
I hold off on the "Navi3.5" change if @stellaraccident's plan is to wait for the gfx1151 patch to land in aotriton upstream. Though note that that PR targets a future aotriton-0.10 and not the current aotriton-0.9.2 release.

@xinyazhang changed aotriton's `main` branch (future 0.10.x) to no longer use  `TARGET_GPUS=Navi32` in https://github.com/ROCm/aotriton/commit/797a9d162f57b88365ab5c8126643518a3fe0e2c

He's still landing major changes for the aotriton-0.10 release, though e.g. https://github.com/ROCm/aotriton/pull/89

Reply:
Let's leave it as-is. I'm not going to nitpick this.